Server/core/CCrashHandler.cpp

```diff
@@ -12,14 +12,14 @@
 #include "StdInc.h"
 #include "CCrashHandler.h"
 #include "version.h"
-#ifdef WIN32
+#ifdef _WIN32
     #include "CExceptionInformation_Impl.h"
 #else
     #include <execinfo.h>
 #endif
 
 // clang-format off
-#ifndef WIN32
+#ifndef _WIN32
```

Comment:
Why change `WIN32` to `_WIN32`?

Reply:
Because `WIN32` might not be defined in some files. `_WIN32` is always defined on Windows. I had issues with `WIN32` not being defined